packages/note/src/react/__stories__/index.story.js

```diff
@@ -0,0 +1,185 @@
+      your painting and get your floor wet. At home you have unlimited time. See
+      there, told you that would be easy. Decide where your cloud lives. Maybe
+      he lives right in here.
+    </p>
+  ),
+  metadata: ['meta first', 'meta second']
+}
+
+storiesOf('Note', module)
+  .addDecorator(fn => <ConstrainWidth>{fn()}</ConstrainWidth>)
+  .add('basic', _ => <NoteWithDefaults />)
+  .add('with linked avatar', _ => (
+    <NoteWithDefaults
+      avatar={
+        <Note.AvatarLink>
+          <a href="#">
+            <Avatar name="Bob Ross" src="//picsum.photos/128" />
+          </a>
+        </Note.AvatarLink>
+      }
+    />
+  ))
+  .add('with linked heading', _ => (
+    <NoteWithDefaults
+      heading={
+        <p>
+          <Link appearance={Link.appearances.subtle}>
+            <a href="#">Bob Ross</a>
+          </Link>
+        </p>
+      }
+    />
+  ))
+  .add('without avatar and author', _ => (
+    <NoteWithDefaults avatar={null} heading={null} />
+  ))
+  .add('with DS Text components', _ => (
+    <NoteWithDefaults
+      heading={
+        <Text.Heading size={Text.Heading.sizes.medium}>
+          <h3>Bob Ross</h3>
+        </Text.Heading>
+      }
+      message={
+        <React.Fragment>
+          <Text.P>
+            Be so very light. Be a gentle whisper. If these lines aren't
+            straight, your water's going to run right out of your painting and
+            get your floor wet. It is a lot of fun.
+          </Text.P>
+
+          <Text.P>
+            So often we avoid running water, and running water is a lot of fun.
+            Just beat the devil out of it. Work that paint. A beautiful little
+            sunset. This painting comes right out of your heart.
+          </Text.P>
+        </React.Fragment>
+      }
+    />
+  ))
+
+storiesOf('Note/metadata', module)
+  .addDecorator(fn => <ConstrainWidth>{fn()}</ConstrainWidth>)
+  .add('with long strings', _ => (
+    <NoteWithDefaults metadata={longStringsMetaData} />
+  ))
+  .add('as a link', _ => (
+    <NoteWithDefaults
+      metadata={[
+        <Link>
+          <a href="#">i'm a link</a>
+        </Link>,
+        <Link appearance={Link.appearances.subtle}>
+          <a href="#">i'm a subtle link</a>
+        </Link>
+      ]}
+    />
+  ))
+
+storiesOf('Note/actions', module)
+  .addDecorator(fn => <ConstrainWidth>{fn()}</ConstrainWidth>)
+  .add('one action', _ => (
+    <NoteWithDefaults
+      actionBar={[
+        <Note.Action icon={<Icon id={Icon.ids.more} />} title="More" />
+      ]}
+    />
+  ))
+  .add('two actions', _ => (
+    <NoteWithDefaults
+      actionBar={[
+        <Note.Action icon={<Icon id={Icon.ids.bookmark} />} title="Bookmark" />,
+        <Note.Action icon={<Icon id={Icon.ids.more} />} title="More" />
+      ]}
+    />
+  ))
+  .add('with long heading', _ => (
+    <NoteWithDefaults
+      heading="This is probably the greatest thing that's ever happened in my life"
```

Comment:
What happens when the heading gets longer than 2 lines?

Reply:
currently it just wraps